pkg/policy/criteria/matchers_test.go

```diff
@@ -66,4 +84,12 @@ func TestStringListMatcher(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, `count([true | some v; v = example[_]; v == "test"]) > 0`, str(body))
 	})
+	t.Run("string", func(t *testing.T) {
```

Comment:
If `is` is now a synonym for `has`, should we also duplicate the `has` test above, and swap in `is` instead of `has`?